Exec/ScalarAdvection/prob.H

```diff
@@ -49,7 +49,7 @@ pc_initdata(
   // Set the density 
   state(i, j, k, Density_comp) = ProbParm::rho_0;
  
-  // Initial temperature -- though we should make sure its consistent with (rho e) above
+  // Initial potential temperature -- though we should make sure it's consistent with (rho e) above
```

Comment:
@pkjha-aero , what `rho e` are you referring to?

Reply:
@ewquon @jmirocha. The original comment was by Ann. I just added 'potential'. But, now that we have our equation for 'rho*theta', this comment about rho*e or rho*E is no longer relevant. So, we can get rid of this.

Reply:
These comments/edits were from days before we had our governing equations and discretizations worked out and documented.